### PR TITLE
renaming the sandbox/ scripts to be lowercase

### PR DESCRIPTION
**Pull request by alameldin**
fixes issue #846

### CODE REVIEW

**Comment by alameldin** [comment]
- [ ] Is it mergeable?
- [ ] Did it pass the tests?
- [ ] If it introduces new functionality in scripts/ is it tested?
  Check for code coverage with `make clean diff-cover`
- [ ] Is it well formatted? Look at `make pep8`, `make diff_pylint_report`,
  `make cppcheck`, and `make doc` output. Use `make format` and manual
  fixing as needed.
- [ ] Did it change the command-line interface? Only additions are allowed
  without a major version increment. Changing file formats also requires a
  major version number increment.
- [ ] Is it documented in the ChangeLog?
  http://en.wikipedia.org/wiki/Changelog#Format
- [ ] Was a spellchecker run on the source code and documentation after
  changes were made?

**Comment by alameldin** [comment]
ready to review

File: ChangeLog [hunk 1]
```diff
@@ -1,3 +1,14 @@
+2015-02-25  Hussien Alameldin  <hussien@msu.edu>
+
+   * sandbox/bloom_count.py: renamed to bloom-count.py
+   * sandbox/bloom_count.pyc: renamed to bloom-count.pyc
```
Comment (ctb) [inline ChangeLog]: the .pyc files are not actually committed in github so you should remove them from the ChangeLog entry.

**Comment by ctb** [comment]
Hah, actually, the goal of the #846 was to change the list of requirements for sandbox files in the docs, not necessarily to change anything in sandbox/. But I don't dislike this set of changes as long as @mr-c agrees.

Main thing TODO - can you edit sandbox/README.rst to reference the new names, and mention that they've been renamed?  Something like

* bloom-count-intersection.py: some description. Renamed from bloom_count_intersection.py in commit f2235bf.

**Comment by mr-c** [comment]
@mr-c is sitting next to @alameldin and approves of the name changes

**Comment by ctb** [comment]
On Wed, Feb 25, 2015 at 02:01:35PM -0800, Michael R. Crusoe wrote:
> @mr-c is sitting next to @alameldin and approves of the name changes

ok, please update README.rst and remove .pyc files from ChangeLog, and
we're good!

**Comment by ctb** [comment]
LGTM. Make it mergeable, please!

**Comment by ctb** [comment]
Thanks!

**Pull request merged**